be/src/io/cache_input_stream.h

```diff
@@ -23,7 +23,7 @@
 
 namespace starrocks::io {
 
-class CacheInputStream final : public SeekableInputStreamWrapper {
+class CacheInputStream : public SeekableInputStreamWrapper {
```

Comment:
Also need a virtual destructor.

Reply:
already existed

Reply:
> already existed

I mean the destructor in `CacheInputStream`, because it has been used as a base class.